### DISCUSSION TITLE
Support deno?

### DISCUSSION THREAD

**Discussion by zuisong** [biomejs/biome#143] [General]
Could be we run biome at deno?
just run
```
deno run -A npm:@biomejs/biome  check . 
```

but now. it does not work . 😭 

node compatibility
https://deno.land/manual@v1.36.4/node/npm_specifiers

**Comment by ematipico** [comment]
It's possible that the members of the core team don't know much about how to set up deno. I am not for example.

If you have experience and would like to help the project, that would be very welcome, and a great contribution!

**Reply by zuisong** [reply]
I am willing to give it a try. I already have some ideas. 😄

**Reply by ematipico** [reply]
Thank you! Feel free to open an issue to track the work

**Reply by zuisong** [reply]
I have created a PR. 😄 
https://github.com/biomejs/biome/pull/151
